packages/component/src/createLoadable.js

```diff
@@ -33,6 +34,23 @@ function createLoadable({ resolve = identity, render, onLoad }) {
       return null
     }
 
+    function resolve(module, props, Loadable) {
+      const Component = options.resolveComponent
```

Comment:
Done. That prevents you from using `resolveComponent` with `loadable.lib`, but then there is no point doing that (since you get the entire module anyway).

Reply:
Also I had to bump up the maximum bundle size